src/theory/quantifiers/sygus_process_conj.h

```diff
@@ -353,8 +353,7 @@ class CegConjectureProcess
                          NodeHashFunction>& free_vars);
   /** for each synth-fun, information that is specific to this conjecture */
   std::map<Node, CegConjectureProcessFun> d_sf_info;
-  /** reference to quantifier engine */
-  QuantifiersEngine* d_qe;
+
```

Comment:
I have downstream changes that are incompatible with this.

Reply:
Right now it is dead code and is triggering warnings in clang. If we need it later, we can add it back.